ci-job-template.yml

```diff
@@ -369,6 +370,14 @@ jobs:
           enabled: false
         service:
           type: LoadBalancer
+        resources:
+          limits:
+            cpu: 2000m
```

Comment:
Only as a reminder: in a real kubernetes cluster we always have to use identical values for limits and requests. Requests is used to find a matching node. If it provides the requested resources, it can be selected. If the application requires more resources (below limit), but the node cannot provide it physically, the pod will be killed. After that it is most likely rescheduled on the same node, finally ending in an endless loop of kill and restart.